src/ring_theory/ideal/operations.lean

```diff
@@ -293,6 +293,12 @@ theorem span_mul_span (S T : set R) : span S * span T =
 submodule.span_smul_span S T
 variables {I J K}
 
+lemma span_mul_span' (T S : set R) : span T * span S = span (T*S) :=
```

Comment:
```suggestion
lemma span_mul_span (T S : set R) : span T * span S = span (T*S) :=
```

Reply:
I believe that there will be namespace errors since the previous theorem has the same name. I was wondering if there was a particular reason to prefer the union of elements over the set product (and if it was significantly harder to tell that if x is in span T*S that there's some t and s when using the set product).

Reply:
Aah, probably historical reasons... anyway, ignore my suggestion and just leave it as it is.